src/state/lottery/getUserLotteryData.ts

```diff
@@ -2,34 +2,35 @@ import { request, gql } from 'graphql-request'
 import { GRAPH_API_LOTTERY } from 'config/constants/endpoints'
 import { LotteryTicket } from 'config/constants/types'
 import { LotteryUserGraphEntity, LotteryResponse, UserRound } from 'state/types'
-import { getRoundIdsArray, fetchMultipleLotteries, hasRoundBeenClaimed, processRawTicketsResponse } from './helpers'
-import { fetchUserTicketsForMultipleRounds } from './fetchUnclaimedUserRewards'
+import { getRoundIdsArray, fetchMultipleLotteries, hasRoundBeenClaimed } from './helpers'
+import { fetchUserTicketsForMultipleRounds } from './getUserTicketsData'
 
 const applyNodeDataToUserGraphResponse = (
-  nodeData: LotteryResponse[],
-  graphData: UserRound[],
-  ticketData: { roundId: string; userTickets: LotteryTicket[] }[],
+  userNodeData: { roundId: string; userTickets: LotteryTicket[] }[],
```

Comment:
Just renaming in this file to better represent where the data is coming from, and what it's about